plugins/inputs/mesos/mesos.go

```diff
@@ -630,6 +650,21 @@ func getMetrics(role Role, group string) []string {
 			"slave/valid_framework_messages",
 			"slave/valid_status_updates",
 		}
+
+		m["overlay"] = []string{
+			"overlay/slave/registering",
```

Comment:
maybe comment that links to https://github.com/dcos/dcos-mesos-modules/blob/master/overlay/agent_metrics.cpp